ical/merger.go

```diff
@@ -27,11 +27,11 @@ func (c *CustomCalender) Merge(url string) (*ics.Calendar, error) {
 	for _, source := range c.source.Info {
```

Comment:
I can't leave a comment higher up... but you can remove the parameter of the function. Thanks